### ISSUE TITLE
ca_summary_view Dataset doesn't include a scheduled refresh

### ISSUE THREAD

**Issue by skarlupka** [aws-samples/aws-cudos-framework-deployment#1056]
When I deployed the Cost Anomaly dashboard the ca_summary_view got successfully added to Quicksight, but the dataset never got a refresh schedule added to it. Is there a way to have the CID-CMD cli add refresh schedules to newly added Quicksight datasets?
<img width="1113" alt="image" src="https://github.com/user-attachments/assets/4ff69ce0-d91e-48e6-a046-51734c2812bf">

**Comment by iakov-aws** [comment]
yes. it is a bug. fixing in https://github.com/aws-samples/aws-cudos-framework-deployment/pull/1057/files

**Comment by iakov-aws** [comment]
fixed. thanks for raising
